### PR TITLE
Updated installation instructions

### PR DESCRIPTION
**Pull request by kheffah**
This addresses issue #705 .

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -49,8 +49,19 @@ HistomicsTK can be used in two ways:
...
+  
+  *To install HistomicsTK using PyPI*::
+  
+  $ pip install histomicstk
+  
+  *To install HistomicsTK from source*::
+  
+  $ git clone https://github.com/DigitalSlideArchive/HistomicsTK/
+  $ cd HistomicsTK/
+  $ python -m pip install scikit-build ninja cython numpy nimfa
```
Comment (manthey) [inline README.rst]: I think this should be the packages in the setup_requires field of setup.py.  That is, `python -m pip install scikit-build Cython cmake setuptools-scm`.  If we need to explicitly install ninja anything else, they should be added to setup_requires.

File: README.rst [hunk 2]
```diff
@@ -49,8 +49,19 @@ HistomicsTK can be used in two ways:
...
+  $ python setup.py develop
```
Comment (manthey) [inline README.rst]: I would prefer to list this as `python -m pip install -e .` (or we could assume that pip is on the command line and just have `pip install -e .`.

File: README.rst (hunk 1 above)
Comment (kheffah) [reply README.rst]: OK, incorporated changes. It turns out `numpy` needs to be explicitly installed so I updated the instructions accordingly.

File: README.rst (hunk 2 above)
Comment (kheffah) [reply README.rst]: OK, done.

File: setup.py [hunk 3]
```diff
@@ -58,12 +58,12 @@ def prerelease_local_scheme(version):
         'Cython>=0.25.2',
         'scikit-build>=0.8.1',
         'cmake>=0.6.0',
+        'numpy>=1.12.1',
     ],
     install_requires=[
         'ctk-cli>=1.5',
         # scientific packages
         'nimfa>=1.3.2',
-        'numpy>=1.12.1',
```
Comment (manthey) [inline setup.py]: numpy needs to remain in the `install_requires` section (in addition to the `setup_requires` section).  The `setup_requires` section is what is necessary before you can install; the `install_requires` section is needed to run it.  If you install from a wheel, you don't have to install the `setup_requires` components, and therefore (I think), installing from a wheel wouldn't install numpy.

Comment (kheffah) [reply setup.py]: Got it, thanks! OK done.

File: README.rst (hunk 2 above)
Comment (satotty) [reply README.rst]: I tried from line 61-64 on command line of my windows computer but when I executed the 64th line, an error came up. I downloaded Microsoft Visual Studio 2019 but it didn’t help anything. What should I do?

The error says:

********************************************************************************
scikit-build could not get a working generator for your system. Aborting build.

Building windows wheels for Python 3.7 requires Microsoft Visual Studio 2017.
Get it with "Visual Studio 2017":

  https://visualstudio.microsoft.com/vs/

********************************************************************************

Comment (kheffah) [reply README.rst]: @satotty I'm not sure how this works on the windows commandline. Ideally, you should be working inside a conda environment to keep things clean. I'd suggest you:

1- Install Anaconda on Widows: https://docs.anaconda.com/anaconda/install/windows/
2- Create a new conda envirnment, say we all it `htk`, and install pip inside it, using something like:
```bash
conda create --name htk
conda install -n htk pip
```
3- Inside the conda command line, activate the environment: `conda activate htk`
4- Follow the same installation instructions provided in the new 'README.rst'

**Pull request merged**